pkg/cms/config/drupal.go

```diff
@@ -125,9 +128,11 @@ func WriteDrushConfig(drushConfig *model.DrushConfig, filePath string) error {
 		return err
 	}
 	// Ensure target directory is writable.
-	os.Chmod(dir, 0755)
+	err = os.Chmod(dir, 0755)
+	util.CheckErr(err)
```

Comment:
it feels like error should just be returned here instead of using `util.CheckErr`, since this function already has an error return value.